mlrun/runtimes/constants.py

```diff
@@ -162,6 +162,7 @@ def terminal_states():
     def error_states():
         return [
             RunStates.error,
+            RunStates.aborting,
```

Comment:
this is not true, aborting is a transitional state, not error state

Reply:
I think that aborting should be considered an error state. Think that if you want to check if a run is in error state you would also want to know if its aborting so its either that or do
```
run.state in (error_states + [aborting])
```
everywhere